### PR TITLE
feature-9077: Missing stats for daily check in scanning returning fro…

### LINKED ISSUE #9077: Missing stats for daily check in scanning returning from API
API should return stats for each day of the number of people that checked in

### PR DESCRIPTION
**Pull request by lthanhhieu**
…m API

Fixes #9077

#### Short description of what this resolves:
- Missing stats for daily check in scanning returning from API

#### Changes proposed in this pull request:
- Missing stats for daily check in scanning returning from API

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

File: migrations/versions/rev-2023-08-17-15:38:43-bce7acfe5a4f_.py
Comment (norbusan) [inline migrations/versions/rev-2023-08-17-15:38:43-bce7acfe5a4f_.py:1]: Whe do we need database migration here

Comment (lthanhhieu) [reply migrations/versions/rev-2023-08-17-15:38:43-bce7acfe5a4f_.py:1]: For fixing issue about the 

 "Multiple Migration Heads"

Comment (norbusan) [reply migrations/versions/rev-2023-08-17-15:38:43-bce7acfe5a4f_.py:1]: Ah, so commit 6a2ecebd1b did introduce multiple migration heads? Very disturbing. That should be tested by CI before merging.

Comment (norbusan) [reply migrations/versions/rev-2023-08-17-15:38:43-bce7acfe5a4f_.py:1]: Indeed indeed, the CI testing was ignored and the PR merged, despite having multiple heads
https://app.circleci.com/pipelines/github/fossasia/open-event-server/7064/workflows/e76c48ce-bfea-4656-ac45-50a8303712df/jobs/26137

**Pull request closed without merging**